github/users.go

```diff
@@ -144,6 +144,22 @@ type UserListOptions struct {
 
 // ListAll lists all GitHub users.
 //
+// To paginate through all users, populate 'Since' with the ID of the last user.
+// Example (warning: this will quickly eat through your GitHub API quota!):
```

Comment:
> However, I'm concerned about making this particular one a real example because then it will actually be run during "go test" which we definitely do not want to do.

Makes sense.

But to clarify, examples only run as part of tests if they include an `// Output:` comment at the end. Otherwise, `go test` simply checks that it compiles, but does not execute it.

> Example functions may include a concluding line comment that begins with "Output:" and is compared with the standard output of the function when the tests are run.

Did you already know that, or is this a new factor?